doc/releases/migration-guide-4.0.rst

```diff
@@ -288,5 +288,12 @@ Shell
 * ``kernel threads`` and ``kernel stacks`` shell command have been renamed to
   ``kernel thread list`` & ``kernel thread stacks``
 
+JWT (JSON Web Token)
+====================
+
+* Default signature library is now PSA Crypto API for both RSA and ECDSA. The
+  newly added :kconfig:option:`CONFIG_JWT_USE_LEGACY` can be used to switch
```

Comment:
```suggestion
* The default signature library is now PSA Crypto API for both RSA and ECDSA. The
  newly-added :kconfig:option:`CONFIG_JWT_USE_LEGACY` can be used to switch
```
Also, I don't know if we can call the PSA Crypto API a library. The library is what implements the API, not the API itself. But maybe I'm being too pedantic here... Do what you want. :slightly_smiling_face: